includes/Admin/Dashboard.php

```diff
@@ -230,19 +231,58 @@ public function redirect_menu_page() {
 	}
 
 	/**
-	 * Preload api requests in the dashboard.
+	 * Preload API requests in the dashboard.
+	 *
+	 * Important: keep in sync with usage & definition in React app.
 	 *
 	 * @since 1.0.0
 	 *
 	 * @return void
 	 */
 	public function load_stories_dashboard() {
-		// Preload common data.
-		// TODO Preload templates.
+		$post_type_object = get_post_type_object( Story_Post_Type::POST_TYPE_SLUG );
+		$rest_base        = $post_type_object instanceof WP_Post_Type && ! empty( $post_type_object->rest_base ) ? $post_type_object->rest_base : $post_type_object->name;
```

Comment:
This what this method is for 

https://github.com/google/web-stories-wp/blob/c755fac29b0c2be27bb1ed6d9ba9dbe54ecdd32a/includes/Traits/Post_Type.php#L39-L56

Reply:
Neat, forgot about this Trait.